packages/ember-template-compiler/tests/basic-usage-test.js

```diff
@@ -65,10 +52,6 @@ moduleFor(
     }
 
     '@test registerPlugin based transforms can be avoided'(assert) {
```

Comment:
I think this whole test is not needed? If there is a test of the basic functionality of plugin elsewhere here, then this test is probably no longer adding coverage. The name of the test here and comments in the body suggest that it was narrowly concerned with the interactions of the two APIs.

Reply:
Good call. Also found another thing to cleanup as well.